docs/DESIGN.md

```diff
@@ -0,0 +1,72 @@
+# Design
+
+* For CSS-in-JS we use styled components if either the styles are more than 2, supposed to be reused at multiple places, need responsive components (using media-queries).
+* rem and pixels both are acceptable.
+* We use [material-ui](https://material-ui.com/) library component and follow [Material Design](https://material.io/design/introduction/#principles) Guidelines
+
+## Colors and Fonts
+
+### Component Colors of Light theme
+
+
+* Application Background Colour: ![#ffffff](https://placehold.it/15/ffffff/000000?text=+) `#ffffff`
+* Message History Background Colour: ![#f5f4f6](https://placehold.it/15/f5f4f6/000000?text=+) `#f5f4f6`
+* Chat Bubble Colour
+    * SUSI.AI Chat bubbles: ![#fffff](https://placehold.it/15/ffffff/000000?text=+) `#ffffff`
+    * User Chat bubbles: ![#e0e0e0](https://placehold.it/15/e0e0e0/000000?text=+) `#e0e0e0`
+* Top Bar Colour: ![#4285f4](https://placehold.it/15/0084ff/000000?text=+) `#4285f4`
+* Buttons Colour: ![#4285f4](https://placehold.it/15/0084ff/000000?text=+) `#4285f4`
+* Search Result Colour: ![#ff5e00](https://placehold.it/15/ff5e00/000000?text=+) `#ff5e00`
+
+* Toggle Colour
+    * thumbOnColor: ![#5ab1fc](https://placehold.it/15/5ab1fc/000000?text=+) `#5ab1fc`
+    * trackOnColor: ![#4285f4](https://placehold.it/15/0084ff/000000?text=+) `#4285f4`
+
+* User Feedback Colour
+    * Thumbs Up Colour
+         **Voted**: ![#1685e5](https://placehold.it/15/1685e5/000000?text=+) `#1685e5`
+         **Unvoted**: ![#90a4ae](https://placehold.it/15/90a4ae/000000?text=+) `#90a4ae`
+    * Thumbs Down Colour-
+    	 **Voted**: ![#d1462f](https://placehold.it/15/d1462f/000000?text=+) `#d1462f`
+         **Unvoted**: ![#90a4ae](https://placehold.it/15/90a4ae/000000?text=+) `#90a4ae`
+
```

Comment:
Please substitute all of the images with images in /docs/images. Don't include images from external sources. Avoid additional dependencies!